src/stack/valgrind.rs

```diff
@@ -80,6 +80,14 @@ cfg_if::cfg_if! {
             );
             result
         }
+    } else if #[cfg(target_arch = "powerpc64")] {
+        type Value = u64;
+
+        // FIXME: look at Valgrind later, use a no-op for now.
```

Comment:
You can find the Valgrind asm sequence for ppc here: https://sourceware.org/git/?p=valgrind.git;a=blob;f=include/valgrind.h.in;h=24a65999851f5b7e771cfb4e3c666c988c101b94;hb=HEAD#l632